docs/EN/index.md

```diff
@@ -1,7 +1,8 @@
 :::{admonition} Work-In-Progress
 :class: caution
 
-**PLEASE NOTE THIS DOCUMENTATION IS STILL BEING WRITTEN BY OUR AWESOME COMMUNITY. UNTIL IT IS FINISHED, PLEASE DO NOT SET UP YOUR TRIO BASED ON THIS DOCUMENTATION ALONE. CONTACT THE COMMUNITY ON DISCORD OR FACEBOOK TO MAKE SURE YOU GET IT RIGHT AND STAY SAFE**
+**GENERAL USERS** This documentation is still under construction during the beta release process. It should not be the main source of information until the main branch of Trio is released. If you have any questions, contact the community on [Discord](https://discord.gg/fCY5svg4) and/or [Facebook](https://www.facebook.com/groups/1351938092206709). Be advised that it is not recommended that general users use the beta version unless they are prepared to update quickly and frequently and report any issues found.
```

Comment:
Discord link is broken.

Reply:
Updated with working, Trio Official-generated, set to never expire with unlimited uses link: https://discord.gg/FnwFEFUwXE